marketplace_standard_app_api/routers/object_storage.py

```diff
@@ -330,3 +332,35 @@ async def delete_dataset(
     https://docs.openstack.org/api-ref/object-store/index.html#delete-object
     """
     raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/mappings",
+    operation_id="listMappings",
+    summary="List all the mappings",
+    tags=["DataSource", "DataSink"],
+    response_model=MappingListResponse,
+    responses={
+        204: {"description": "No mappings found."},
+    },
+)
+async def list_mappings(
+    limit: int = 100, offset: int = 0
+) -> Union[MappingListResponse, Response]:
+    """List all mappings."""
+    raise HTTPException(status_code=501, detail="Not implemented.")
+
+
+@router.get(
+    "/mappings/{mapping_id}",
+    operation_id="getMapping",
```

Comment:
Suggestion: `getSemanticMapping`

Reply:
Same as above, endpoint name and operation id should be consistent.

Reply:
See here: https://github.com/materials-marketplace/standard-app-api/pull/34#discussion_r925262201

Reply:
This would potentially need to be changed to `/semanticMappings/{semantic_mapping_id}` and `getSemanticMapping`.